### PR TITLE
Add basic jinja and ruby support

### PR DESCRIPTION
**Pull request by Bozydarek**
Add basic ruby support
Also changes in README and CHANGELOG files

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "jinja2",
     "django",
     "markdown",
+    "ruby",
```
Comment (samuelcolvin) [inline package.json]: I think from memory it's already at the limit of the number of tags you can add, I guess you'll have to remove this.

File: package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "jinjahtml",
   "displayName": "Better Jinja",
   "description": "Syntax highlighting for jinja(2) including HTML, Markdown and YAML templates",
-  "version": "0.3.0",
+  "version": "0.4.0",
```
Comment (samuelcolvin) [inline package.json]: can you change the description to `Syntax highlighting for jinja(2) including HTML, Markdown, YAML and ruby templates`

**Comment by samuelcolvin** [comment]
awesome, thank you.

**Comment by Bozydarek** [comment]
Sure, no problem :wink:

**Comment by samuelcolvin** [comment]
new version released.

**Pull request merged**